README.md

```diff
@@ -89,6 +89,19 @@ npm test
 ```bash
 docker compose exec app npm run test-integrations
 ```
+If you'd like to be able to run and debug integration tests directly in your IDE, run the following command (note that it may prompt you for your `sudo` password to modify your `/etc/hosts` file):
+
+```bash
+npm run test-setup
```

Comment:
one blocker i have for this PR - add a comment that tells me exactly what this is going to do to my machine and why 😄